app/models/business.rb

```diff
@@ -13,13 +13,39 @@ class Business < UuidApplicationRecord
 
   accepts_nested_attributes_for :children
 
+  QRIS_RATINGS = %w[
+    not_rated
+    step_one
+    step_two
+    step_three
+    step_four
+    step_five
+    gold
+    silver
+    bronze
+  ].freeze
+
   validates :active, inclusion: { in: [true, false] }
+  validates :qris_rating, inclusion: { in: QRIS_RATINGS }, allow_nil: true
   validates :name, presence: true, uniqueness: { scope: :user_id }
   validates :county, presence: true
   validates :zipcode, presence: true
 
   scope :active, -> { where(active: true) }
 
+  def ne_qris_bump
+    case qris_rating
+    when 'step_three'
+      accredited ? 1.0 : 1.05
+    when 'step_four'
+      accredited ? 1.05 : 1.1025
+    when 'step_five'
+      accredited ? 1.1025 : 1.1576
```

Comment:
Small precision edit.  These numbers come compounding formula (1+.05)^N.  So its 0 - 1.0, 1 - 1.05, 2 - 1.1025, 3 - 1.157625.  Should we either calculate or use precise decimal so we don't have a weird off-by-one-cent issue?

Also relatedly, I think we should then round to the nearest cent to use the rates b/c NE DHHS probably does (I believe this b/c they send providers a published rate rounded to the nearest cent)